vcpkg/concepts/ports.md

```diff
@@ -0,0 +1,60 @@
+---
+title: Ports
+description: Ports as a vcpkg concept
+author: JavierMatosD
+ms.author: javiermat
+ms.date: 09/21/2023
+ms.prod: vcpkg
+ms.topic: concept-article
+---
+
+# Concept: Ports
+
+A port in vcpkg is a versioned recipe that produces a set of files. Executing a port affects the installation graph by producing new header or binary files. Ports may have dependencies on other ports and may require additional dependencies when certain features are enabled, these dependencies must be reflected in the ports metadata. At a minimum, a port contains:
+
+* Metadata about the package: name, version, supported features, dependencies, etc.
+* Instructions to acquire, build if necessary, and install the package.
+
+Additionally, ports may include:
+
+* Patches to apply to the package's source code
+* Instructions to integrate with your project's build system
+
+## Portfile (`portfile.cmake`)
+
+All ports must contain a portfile (a file named `portfile.cmake`). A portfile is a script that contains instructions on how to build and install a package within the vcpkg environment. Some common tasks performed by portfiles are:
+
+* Download a package's source code.
+* Build the package using the upstream build system.
+* Copy the build output to the vcpkg installation tree.
+* Install build system integration files.
+
+## vcpkg.json
+
+The `vcpkg.json` file describes metadata about the package contained in the vcpkg port. This file defines attributes like the package’s name, version, description, dependencies, supported features and platforms, etc.
+See the [`vcpkg.json`](../reference/vcpkg-json.md) file reference to learn more.
+
+## Patch files
+
+In certain scenarios, a port might include patch files. These files carry modifications to the library's source code, which are essential to either fix existing issues or to maintain compatibility with other components. Before the build process, patches are applied as part of the portfile execution steps, aiding in a seamless and error-free library integration.
+
+## Port categories
```

Comment:
I think it would be good to add a paragraph here saying something that port categories aren't really a 'product feature'; they are something maintainers say colloquially to each other but there's no reason that a port can't be several or none of these.

This is kind of blurring the line between documentation and maintainer notes; I think it's OK here though